SeaPkg/Core/Runtime/SeaResponderReport.c

```diff
@@ -105,12 +104,11 @@ Range1InsideRange2 (
 EFI_STATUS
 EFIAPI
 VerifyAndHashImage (
-  IN  UINTN               ImageBase,
-  IN  UINT64              ImageSize,
-  IN  UINT64              AuxFileBase,
-  IN  UINT64              AuxFileLength,
-  IN  UINT64              PageTableBase,
-  OUT TPML_DIGEST_VALUES  *DigestList
+  IN  UINTN                         ImageBase,

```

Comment:
Something else that sticks out about the function signature is that `ImageBase` is `UINTN` and `PageTableBase` is `UINT64` though they are both base addresses. Is there a reason both base addresses don't use the same type? It would be clearer to use `EFI_PHYSICAL_ADDRESS` which is a `UINT64`.

Reply:
Sure, I can update this to `EFI_PHYSICAL_ADDRESS` so long as @kuqin12 does not have a specific reason these are different

Reply:
Updated in the last commit. Can drop if need be, depending on kuqin12.

Reply:
I think updating to efi_physical_address makes sense! Thanks for fixing it!